core/controllers/profile.py

```diff
@@ -83,6 +86,9 @@ def get(self, username):
                 utils.get_time_in_millisecs(user_settings.first_contribution_datetime)
                 if user_settings.first_contribution_datetime else None),
             'profile_picture_data_url': user_settings.profile_picture_data_url,
+            'created_explorations_count': len(user_contributions.created_explorations),
```

Comment:
Yes, I think so. Only pass to the frontend what you want to be displayed in the frontend.